### ISSUE TITLE
Upgrade to Kubernetes 1.10?

### ISSUE THREAD

**Issue by adarshaj** [coreos/tectonic-forum#312]
## Feature Request
Upgrade to Kubernetes 1.10

What is the release plan for next tectonic version with k8s 1.10?
### Environment

What hardware/cloud provider/hypervisor is being used with Tectonic?
bare metal

**Comment by rushins** [comment]
Hi  , i just wanted to share.

I have requested the same to tectonic  community while ago on another thread, however there was  no reply to that since from August . I think due to Tectonic (Coreos) acquistion by Redhat and Redhat acquired by IBM. This will lead to some uncertinity until we hear back from the product management of COREOS/tectonic/Redhat ?

**Comment by adarshaj** [comment]
Agree, but we've moved on. Tectonic worked great for us, but needed some of new features for our deployments, so had to move on to something that's better maintained and is in sync with kubernetes upstream (though without the one-click upgrades).

**Comment by rushins** [comment]
you can check SUSE CAASP which is opensource and works nicely with ISO image bootstrape of kubernetes cluster anywhere whether baremetal or virtual ? just download and bootstarp ISO after that you can decide the master/worker cluster through a nice workflow application..
here is the link.
https://www.suse.com/products/caas-platform/

**Comment by adarshaj** [comment]
I think those who are acquainted with tectonic might find https://typhoon.psdn.io/ to be the most comfortable to switch to without much of an extra learning curve. It has the same tectonic like terraform support (infact the author is same).
